tests/end-to-end/simple/App.cs

```diff
@@ -0,0 +1,203 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.S3;
+using Amazon.CDK.AWS.SQS;
+using Constructs;
+using System.Collections.Generic;
+
+namespace Com.Acme.Test.Simple
+{
+    public class SimpleStackProps : StackProps
+    {
+        /// <summary>
+        /// The prefix for the bucket name
+        /// </summary>
```

Comment:
I was gonna ask if we could use multi line comments here, but this seems to be the standard which is horrifying https://www.w3schools.com/cs/cs_comments.php#:~:text=C%23%20Multi%2Dline%20Comments,will%20be%20ignored%20by%20C%23.

Reply:
Yes I went to a lot of trouble to find examples in the wild to confirm that this is the c# style.